src/ui/camera.js

```diff
@@ -295,41 +306,90 @@ class Camera extends Evented {
 
     /**
      * Pans and zooms the map to contain its visible area within the specified geographical bounds.
+     * This function will also reset the map's bearing to 0 if bearing is nonzero.
      *
      * @memberof Map#
      * @param {LngLatBoundsLike} bounds Center these bounds in the viewport and use the highest
      *      zoom level up to and including `Map#getMaxZoom()` that fits them in the viewport.
-     * @param {Object} [options]
+     * @param {AnimationOptions | CameraOptions | PaddingOptions} [options]
```

Comment:
The way I read this, it makes me think that `top`, `bottom`, etc. are properties that can go _directly_ on `options`.  Since these should actually be sub-properties of `options.padding`, should this be `@param {number|PaddingOptions} [options.padding=0] The amount of padding to allow around the given bounds.` (or something less stilted-sounding heh)